### DISCUSSION TITLE
Commands not putting to chlorinator

### DISCUSSION THREAD

**Discussion by ar26pt2** [tagyoureit/nodejs-poolController#435] [General]
Great product! I love it.  But alas I realize i cant control my chlorinator yet.  I'm using dashPanel, Pentair ic40 on EasyTouch2 4P.  It seems to receive salt level from the OCP but does not accept an update of Pool SetPoint level or SuperChlor button from the dashPanel.  (Note it seems like the dashPanel keeps the SetPoint I enter, but it never changes on the OCP).  SuperChlor just sticks on Initializing...  I saw in the poolstate.json that the pool's actual setpoint at the panel is correctly stored as "targetoutput".

I tried removing the chlorinator then adding again, Pi reboot.  Everything else works very well.  Attached is console and packet log of my attempts today.    Any help is welcome! ar

[packetLog(2022-05-07_17-02-23).log](https://github.com/tagyoureit/nodejs-poolController/files/8646320/packetLog.2022-05-07_17-02-23.log)
[consoleLog(2022-05-07_17-02-23).log](https://github.com/tagyoureit/nodejs-poolController/files/8646321/consoleLog.2022-05-07_17-02-23.log)

**Comment by tagyoureit** [comment]
Hi,
First, can you please upload a full [packet capture](https://github.com/tagyoureit/nodejs-poolController/wiki/How-to-capture-all-packets-for-issue-resolution) so we can see your configuration.  

From the short captures that you updated, there is no communication at all to or from the Chlorinator.  Not from njsPC and not from EasyTouch.  This could be because you don't have all the logging turned on properly.

Can you confirm that you have the chlorinator working and can change the setpoint from EasyTouch?  njsPC should automatically pick up the chlorinator based on the OCP (outdoor control panel) configuration.  You should not need to add it manually.

**Comment by ar26pt2** [comment]
[replay (6).zip](https://github.com/tagyoureit/nodejs-poolController/files/8647136/replay.6.zip)

Thank you for the reply!  Ok, well good news.. **It seems like things are working**.  I did a power cycle of the pool equipment.  I did a reboot of the pi. I removed the controller from dashpanel.  I rebooted pi.  I waited.  when dashpanel reloaded the chlorinator was still **not loaded automatically, so I added it manually again**.  BUT IT SEEMS TO BE COMMUNICATING and WORKING now.  _(If you read the logs you'll note my salt level is way high, its because i just added salt and it is near the main pool drain so its artificially reading high this weekend.)_

Thank you for all the help and great tool! ar
